packages/cli/test/integration/shared/db/evaluation.ts

```diff
@@ -0,0 +1,68 @@
+import { TestRunRepository, TestCaseExecutionRepository } from '@n8n/db';
+import type { TestRun } from '@n8n/db';
+import type { TestCaseExecution } from '@n8n/db';
+import type { AggregatedTestRunMetrics } from '@n8n/db';
+import type { TestCaseExecutionErrorCode, TestRunErrorCode } from '@n8n/db';
+import { Container } from '@n8n/di';
+import type { IDataObject } from 'n8n-workflow';
+
+/**
+ * Creates a test run for a workflow
+ */
+export const createTestRun = async (
```

Comment:
nitpick/optional - I find that the word `test` is confusing. I am not sure if I am looking at production or test code. Maybe we can rename these to `createMockTestRun` and `createMockTestCaseExecution`. 

Especially as we use `testRunRepository.createTestRun` directly in the tests as well.

I can imagine this will be a point of confusion for everyone who does not have context on the evaluation feature, for example seeing a file called `test-runner`

Reply:
I think we can move towards using term `evaluation` instead of `test` in this feature, but it will greatly complicate merging this if rename now